<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Endereco</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Sexo</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLineEdit" name="line_Phone"/>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLineEdit" name="line_Name"/>
    </item>
    <item row="9" column="2" colspan="3">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="3">
     <widget class="QLineEdit" name="line_Address"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QSpinBox" name="spin_max"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Cidade</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="line_City"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Telefone</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Random - Me!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="3">
     <widget class="QLineEdit" name="line_Gender"/>
    </item>
    <item row="6" column="3">
     <widget class="QSpinBox" name="spin_min"/>
    </item>
    <item row="5" column="3" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" alignment="Qt::AlignHCenter|Qt::AlignBottom">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Max</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>543</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
